### DISCUSSION TITLE
Scoop: Add alias to binary

### DISCUSSION THREAD

**Discussion by runar-rkmedia** [goreleaser/goreleaser#3047] [Q&A] [answered]
I have a cli-tool which I wish to publish on scoop, and goreleaser works great here. However, I don't see a way to add an [alias under the bins-option for scoop](https://scoop-docs.vercel.app/docs/concepts/App-Manifests.html#a-simple-example).

Is there a possible way to do so? I looked through the source-code for `scoop.go` and from what I can tell it seems to reference the an `Extra`-field, but it does not seem to read it for `bins`.

Any tips?

**Comment by caarlos0** [comment, marked as answer]
I think this option is not supported on goreleaser... feel free to open a pr though
